internal/testing/gouging_test.go

```diff
@@ -132,26 +132,4 @@ func TestGouging(t *testing.T) {
 	if err := w.UploadObject(ctx, bytes.NewReader(data), name); err == nil {
 		t.Fatal("expected upload to fail")
 	}
-
-	// update all host settings so they're gouging
-	for _, h := range hosts {
-		settings := h.settings.Settings()
-		settings.MinEgressPrice = types.Siacoins(1)
-		if err := h.UpdateSettings(settings); err != nil {
-			t.Fatal(err)
-		}
-		// assert it was removed from the contract set
-		if err := waitForHostRemoval(h.PublicKey()); err != nil {
-			t.Fatal(err)
-		}
-	}
-
-	// download the data - should fail
-	buffer.Reset()
-	if err := w.DownloadObject(ctx, &buffer, name); err == nil {
```

Comment:
Shouldn't this still be valid? Downloads failing if the hosts are gouging?

Reply:
This turned out to be a huge PITA. The price table caching in the worker prevents the host from being seen as price gouging. I opened a PR in `hostd` to lower the default value and make it configurable. It should work but will require a sleep 🤷‍♂️ .